src/frontend/src/states/SettingsState.tsx

```diff
@@ -28,6 +28,10 @@ export const useGlobalSettingsState = create<SettingsStateProps>(
     lookup: {},
     endpoint: ApiEndpoints.settings_global_list,
     fetchSettings: async () => {
+      if (!useServerApiState.getState().authenticated) {
```

Comment:
Question: Why use it in this form like in the rest of the code base?
```suggestion
      const [authenticated] = useServerApiState((state) => [state.authenticated]); 
      if (!authenticated) {
```

Reply:
Because Hooks can only be invoked in components, unconditionally on every render.